Apps/W1/Shopify/app/src/Order handling/Codeunits/ShpfyOrderMapping.Codeunit.al

```diff
@@ -247,6 +249,26 @@ codeunit 30163 "Shpfy Order Mapping"
         end;
     end;
 
+    local procedure MapShippingAgent(var OrderHeader: Record "Shpfy Order Header")
+    var
+        OrderShippingCharges: Record "Shpfy Order Shipping Charges";
+        ShipmentMethodMapping: Record "Shpfy Shipment Method Mapping";
+        IsHandled: Boolean;
+    begin
+        if OrderHeader."Shipping Agent Code" = '' then begin
+            OrderEvents.OnBeforeMapShipmentAgent(OrderHeader, IsHandled);
+            if not IsHandled then begin
+                OrderShippingCharges.SetRange("Shopify Order Id", OrderHeader."Shopify Order Id");
+                if OrderShippingCharges.FindFirst() then
```

Comment:
There may be more than one shipping charge per order (add shipping charge, get payment and add another one)
You don't need to do anything, only commenting to document limitation (on us). We already have the same with shipping method